modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -166,7 +166,7 @@ spec:
 <9> For Intel Columbiaville 800 Series NICs, ensure `boundary_clock_jbod` is set to `0`. For Intel Fortville X710 Series NICs, ensure `boundary_clock_jbod` is set to `1`.
 <10> Specify system config options for the `phc2sys` service. If this field is empty the PTP Operator does not start the `phc2sys` service.
 <11> Scheduling policy for ptp4l and phc2sys processes. Default value is `SCHED_OTHER`. Use `SCHED_FIFO` on systems that support FIFO scheduling.
-<12> Integer value from 1-65 used to set FIFO priority for `ptp4l` and `phc2sys` processes. Required if `SCHED_FIFO` is set for `ptpSchedulingPolicy`.
+<12> Integer value from 1-65 used to set FIFO priority for `ptp4l` and `phc2sys` processes. Required if `SCHED_FIFO` is set for `ptpSchedulingPolicy`. This value is unusued otherwise.
```

Comment:
Suggest clarifying when the field is not used.

```suggestion
<12> Integer value from 1-65 used to set FIFO priority for `ptp4l` and `phc2sys` processes when `ptpSchedulingPolicy` is set to `SCHED_FIFO`. The `ptpSchedulingPriority` field is not used when `ptpSchedulingPolicy` is set to `SCHED_OTHER`.
```

Reply:
It is explained here: https://51637--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp.html#cnf-configuring-fifo-priority-scheduling-for-ptp_using-ptp

I can add it in additional info after each section.